pkg/kubectl/cmd/resource/get.go

```diff
@@ -320,145 +345,51 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 		objs[ix] = infos[ix].Object
 	}
 
-	sorting, err := cmd.Flags().GetString("sort-by")
+	showKind := options.ShowKind || resource.MultipleTypesRequested(args) || cmdutil.MustPrintWithKinds(objs, infos, nil)
+	options.HumanPrintFlags.ShowLabels = &showKind
```

Comment:
ShowKind gets set through a flag, but we end up overriding it here anyway, since it appears that we want to show it regardless of what the flag says when there is more than one resource kind

Reply:
This looks wrong.  Setting showlabels based the kind value?

Reply:
Thanks, meant to set `options.HumanPrintFlags.ShowKind = &showKind`. Updated